df.items.xml

```diff
@@ -694,6 +695,7 @@
 
         <int32_t name='unk_1' init-value='-1' since='v0.34.01'/>
         <int32_t name='temp_updated_frame' init-value='-1'/>
+        <int32_t/>
```

Comment:
Why was this field added? It seems to be the cause of my issues. I think it only exists as the base class padding with msvc on 64 bits.

Reply:
Hmm, do we have a way to represent that in df-structures?

Reply:
It does not need to be represented. It is generated by the compiler.

Reply:
Actually, looking at your lua output above, it looks like my added field *fixed* the problem you were seeing.

Reply:
The output was made including this change. `anon_1` is this field, but the value 38 is expected for mat_type.

Reply:
You did not answer my question. What was this fixing? I cannot simply remove it if it will break something else.

Reply:
disclaimer: not an expert
@BenLubar and @cvuchener we have a way to represent it in df-structures and yes it needs to be represented as generated structure is different.E.g:
`
struct A{
int x;
int y;
};
`
and
`
struct A
{
struct {int x;};
int y;
};
`
**might** be laid out differently.

Reply:
> `struct A { struct {int x;}; int y; };`

This is exactly what this case is **not**. GCC does not add padding at the end of the base class when they are included in a derived class, unlike POD types.

Reply:
Never mind, I just saw that it was reverted in a6477749abeaddf8c5edcd5174874892b177cce2.